contracts/RemovalUtils.sol

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.13;
+
+import "@openzeppelin/contracts-upgradeable/utils/math/MathUpgradeable.sol";
+
+struct UnpackedRemovalIdV0 {
+  uint256 idVersion;
+  uint256 methodology;
+  uint256 methodologyVersion;
+  uint256 vintage;
+  string country;
+  string admin1;
+  address supplierAddress;
+  uint256 subIdentifier;
+}
+
+uint256 constant _BITS_PER_BYTE = 8;
```

Comment:
are we using this leading underscore, all caps constant style elsewhere?

Reply:
all caps yes... at least I see that for constants in `LockedNORI`.  leading underscore just indicating the privacy of the variable

Reply:
confirmed that this is what OZ uses for internal constants.  going to mark all of those constants as internal, which is the default visibility level for all state variables

Reply:
oh wait, since these are top level variables, they're not even part of a contract... so in that sense that are private in that they're not going to be visible to anything but whatever's defined in this file, into which they will be inlined as constants.  so i'm just going to leave this naming convention